pino.js

```diff
@@ -138,10 +138,14 @@ function pino (...args) {
     [stringifySym]: stringify,
     [formattersSym]: allFormatters
   })
-  const chindings = base === null ? '' : (name === undefined)
-    ? coreChindings(base) : coreChindings(Object.assign({}, base, { name }))
+  const chindings = base === null
+    ? ''
+    : (name === undefined)
+        ? coreChindings(base)
+        : coreChindings(Object.assign({}, base, { name }))
```

Comment:
Ugh. I didn't realize we had nested ternaries in this code. I'd rather eliminate the nesting. The indentation helps, but it's still difficult to reason through.

Reply:
Done